src/org/opendatakit/briefcase/model/OdkCollectFormDefinition.java

```diff
@@ -117,7 +97,8 @@ public boolean equals(Object obj) {
       String id = getFormId();
       String versionString = getVersionString();
 
-      return (id.equals(lf.getFormId()) && ((versionString == null) ? (lf.getVersionString() == null)
+      return (id.equals(lf.getFormId()) && ((versionString == null)
```

Comment:
I’d remove the outermost, redundant, pair of parentheses. When I see `return (expression)` it suggests the programmer thinks that return is a function and not a statement.